data/json/items/gun/762x25.json

```diff
@@ -28,5 +28,30 @@
     "magazines": [
       [ "762x25", [ "ppshmag", "ppshdrum" ] ]
     ]
+  },
+  {
+    "id": "tokarev",
+    "type": "GUN",
+    "symbol": "(",
+    "color": "dark_gray",
+    "name": "Tokarev TT-30",
+    "name_plural": "Tokarev TT-30",
+    "description": "The Norinco manufactured Tokarev TT-30 is the standard sidearm of the Chinese military, it does not see extensive use outside of China.",
+    "price": 140000, "//" : "And it's therefore expensive.",
+    "material": ["steel", "plastic"],
+    "skill": "pistol",
+    "ammo": "762x25",
+    "weight": 798,
+    "volume": 2,
+    "bashing": 10,
+    "to_hit": -2,
+    "dispersion": 225,
+    "sight_dispersion": 90,
+    "aim_speed" : 4,
+    "recoil": 60,
+    "durability": 6,
+    "magazine_well": 1,
```

Comment:
Wikipedia says the Chinese version is banned due to import restrictions, but the Russian and Czech versions are unaffected.